tasks/task03/ Word_stats.py

```diff
@@ -0,0 +1,16 @@
+import re
+texts = ["Hello, World!", "The world is mine", "Hello, how are you?"]
+dictw = {}
+i = 0
+for istr in texts:
+    for iword in re.split(r'[:; ,*.!?\n]+',istr.lower()):
+        if iword == "":
+            continue
+        if dictw.get(iword, "e404") == "e404":
```

Comment:
In Python, use None to denote absence of a value. dict.get(key) will return None by default if such key is not present.

In your case you actually do not need the value, you only check if it is present or not.
In this case, use 
if key not in dict